modules/apps/27-interchain-accounts/host/keeper/relay_test.go

```diff
@@ -494,11 +529,12 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 
 				txResponse, err := suite.chainB.GetSimApp().ICAHostKeeper.OnRecvPacket(suite.chainB.GetContext(), packet)
 
-				if tc.expPass {
-					suite.Require().NoError(err)
+				expPass := tc.expErr == nil
+				if expPass {
+					suite.Require().Nil(err)
```

Comment:
just curious, why the change from `NoError` to `Nil`?

Reply:
I'm pretty sure `tc.expErr == nil` moved me in that direction as I was reading it. Looking at others I do see `NoError` preferred so I can swap to that.